pyanaconda/ui/gui/__init__.py

```diff
@@ -783,7 +783,7 @@ def _instantiateAction(self, actionClass):
         return obj
 
     def run(self):
-        (success, args) = Gtk.init_check(None)
+        (success, _args) = Gtk.init_check(None)
```

Comment:
Since `_args` is not used anywhere else, maybe there could be `success, _ = Gtk.init_check(None)`.